apim/3.x/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents all notable changes to [Gravitee.io API Management 3.x](https://github.com/gravitee-io/helm-charts/tree/master/apim/3.x) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### 3.1.34
+
+- APIM: disable old classloader to enable the new one by default (since v3.15) 
```

Comment:
To avoid confusion
```suggestion
- APIM: disable old classloader to enable the new one by default (since APIM v3.15) 
```

Reply:
It is already said at the beginning no ? `"APIM: ...."`

Reply:
for me there is no need to precise `APIM` as the changelog is specific to APIM : https://artifacthub.io/packages/helm/graviteeio/apim3?modal=changelog